uicomponent-compose/main/src/main/java/io/github/droidkaigi/feeder/AppContent.kt

```diff
@@ -168,6 +175,50 @@ fun AppContent(
                     },
                 )
             }
+            composable(
+                route = "$NOTIFICATION_PATH{notification}",
+                deepLinks = listOf(
+                    navDeepLink {
+                        uriPattern = "$deepLinkUri/$NOTIFICATION_PATH{notification}"
+                    }
+                ),
+                arguments = listOf(
+                    navArgument("notification") {
+                        type = NavType.StringType
+                    }
+                )
+            ) { backStackEntry ->
+                val routePath = rememberRoutePath(
+                    backStackEntry.arguments?.getString("notification")
+                        ?: OtherTab.AboutThisApp.routePath
+                )
+                val selectedTab = NotificationTab.ofRoutePath(routePath.value)
+                drawerContentState.onSelectDrawerContent(selectedTab)
+                NotificationScreen(
+                    onNavigationIconClick = onNavigationIconClick,
+                    onDetailClick = { notificationItemId ->
+                        actions.onSelectNotificationDetail(notificationItemId)
+                    }
+                )
+            }
+            composable(
+                route = "$NOTIFICATION_DETAIL_PATH{id}",
```

Comment:
Thanks for making the detail screen. It might be enough if we could navigate to the link as well as the feed. No need to address that in this PR!